src/main/java/lottery/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package lottery;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));

```

Comment:
my previous comment is not fixed 

`100 should be a constant field

also, we can declare new ColorSupplier() and new Random() as fields. so that we don't create a new object each time the method is called`